src/api/package/docs.ts

```diff
@@ -35,3 +36,42 @@ export const fetchMarkdown = async (
   }
   return response.text();
 };
+
+/**
+ * Fetch JSON docs of a specific package from the backend.
+ */
+export const fetchJsonDocs = async (
+  name: string,
+  version: string,
+  language: string,
+  scope?: string,
+  submodule?: string
+): Promise<Schema> => {
+  const docsSuffix = `/docs-${
```

Comment:
Nit: could you generalize this to a local `getDocsSuffix` helper so we can re-use it in both `fetchMarkdownDocs` and `fetchJsonDocs`?